### ISSUE TITLE
How do I extend OAuth2 Scheme?

### ISSUE THREAD

**Issue by damafeez** [nuxt-community/auth-module#780]
I am trying to create a new scheme that extends off of `OAuth2`. Like so:
```
import OAuth2 from '@nuxtjs/auth/lib/schemes/oauth2'

export default class newScheme extends OAuth2 {...
```

I get this error ```ERROR in ./node_modules/@nuxtjs/auth/lib/schemes/oauth2.js
Module not found: Error: Can't resolve '../utilities' in '/Users/afeezawoyemi/workspace/P/logistics-config-ui/node_modules/@nuxtjs/auth/lib/schemes'```

It appears the `utilities` module is indeed not present in the specified relative path, what is the right way to extend `OAuth2` scheme?

#### STEPS TO REPRODUCE
- create a custom scheme
- extend OAuth2 scheme like so:
```
import OAuth2 from '@nuxtjs/auth/lib/schemes/oauth2'

export default class newScheme extends OAuth2 {...
```
- attempt to use your new scheme by adding it to `nuxt.config.js`

#### WHAT HAPPENS
```
ERROR in ./node_modules/@nuxtjs/auth/lib/schemes/oauth2.js
Module not found: Error: Can't resolve '../utilities' in '/Users/afeezawoyemi/workspace/P/logistics-config-ui/node_modules/@nuxtjs/auth/lib/schemes'
```

#### WHAT SHOULD HAPPEN
New scheme should work without errors

PS: 
Thank y'all for your contributions to the Nuxt community.
May be related to [#176](https://github.com/nuxt-community/auth-module/issues/176)

**Comment by NickBolles** [comment]
Might be fixed by https://github.com/nuxt-community/auth-module/pull/774? @timrchavez @JoaoPedroAS51

**Comment by damafeez** [comment]
#### TL/DR? WHAT'S THE RIGHT WAY TO USE/REFERENCE THE MORE UP TO DATE BRANCHES IN MY CODE?

Digging a little deeper, I see the issue originated from this [line](https://github.com/nuxt-community/auth-module/blob/a9d3f6c302e37c3149cf80cb912a1c3faa8f9d66/lib/schemes/oauth2.js#L1) of the [4.9.1 release](https://github.com/nuxt-community/auth-module/releases/tag/v4.9.1) (the latest official release, which I'm also currently using).

```import { encodeQuery, parseQuery, addTokenPrefix } from '../utilities'```
whereas there's no `utilities` directory/file in the specified relative path (`utilities` is defined in [lib/core/utilities.js](https://github.com/nuxt-community/auth-module/blob/a9d3f6c302e37c3149cf80cb912a1c3faa8f9d66/lib/core/utilities.js), which doesn't match the specified path). 

Only seems to freak out when `oauth2` is imported into another module.

I see there's been a handful of commits that should have fixed that, so I think the question should be WHAT'S THE RIGHT WAY TO USE THESE MORE UP TO DATE BRANCHES, or how soon will there be another official release?

**Comment by gorenburg** [comment]
Bumped into this issue as well. Probably path to methods should be updated to fix the issue

**Comment by damafeez** [comment]
I switched to the [dev branch](https://dev.auth.nuxtjs.org/guide/setup), that solves my issue.
I don't know if it's worth raising a pr to fix this on master? Otherwise, I'm willing to close this since the dev branch is more updated and should fix issues like this mostly.

**Comment by gorenburg** [comment]
@damafeez yes, i see they've moved `utils` to the proper location and it should work there. do you know any ETA when dev would be merged into master?

p.s. i think we can close PR with mark that it's fixed in dev branch for future reference

**Comment by damafeez** [comment]
@gorenburg re ETA, I've been wondering also. `master`'s been behind for so long, I wonder if there's any plan to merge them soon. Anyway, I'll just stick to the dev branch for now.
